### PR TITLE
Update for latest CDAP

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>groupId</groupId>
-    <artifactId>Kinesis-sparkStreamingSource</artifactId>
-    <version>1.0.0</version>
+    <artifactId>kinesis-streaming-source</artifactId>
```
Comment (albertshau) [inline pom.xml]: this is a backwards incompatible change.

File: pom.xml [hunk 2]
```diff
@@ -64,15 +64,15 @@
...
+        <cdap.version>4.3.0</cdap.version>
         <hadoop.version>2.3.0</hadoop.version>
         <junit.version>4.11</junit.version>
         <hydrator.version>1.6.0</hydrator.version>
         <spark.version>1.6.2</spark.version>
         <widgets.dir>widgets</widgets.dir>
         <docs.dir>docs</docs.dir>
         <app.parents>
-            system:cdap-etl-batch[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
+            system:cdap-data-pipeline[4.0.0,10.0.0-SNAPSHOT),system:cdap-data-streams[4.0.0,10.0.0-SNAPSHOT)
```
Comment (albertshau) [inline pom.xml]: Does this only contain the streaming source? If so, cdap-data-pipeline should not be a parent either.

File: pom.xml (hunk 1 above)
Comment (albertshau) [reply pom.xml]: though I agree that the current name isn't very good, just making sure the implications are clear.

Comment (bdmogal) [reply pom.xml]: thought about that - how do we handle these? Should I bump the version to 2.0.0?

too bad we didn't catch this during code review.

Comment (bdmogal) [reply pom.xml]: have reverted the artifact name change right now.

File: pom.xml (hunk 2 above)
Comment (bdmogal) [reply pom.xml]: fixed

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**